### PR TITLE
Adding sounds and sound features

### PR DESCRIPTION
**Pull request by secretrobotron**
Added a few cool sounds from @DevonBird 🥇 

@DevonBird not sure about how to organize a couple of them though. What's the difference between Low/Mid/High? Just volume?

### CODE REVIEW

**Comment by secretrobotron** [comment]
> Low/Mid/High

Ok, figured it out :). There's a differentiation for the mines now.

**Comment by secretrobotron** [comment]
@flukeout how's this feel?

**Comment by flukeout** [comment]
@secretrobotron Yeah good! Like I mentioned earlier, we'll need to adjust the volume of all the sounds so they're more cohesive. Also, I think you made it so only mines spawn?

File: battle-pong/src/js/settings.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ window.Settings = {
   clearSavedControlSettings : true,
   showIntro : false,
   debug : true,
-  powerUpTypes : ["mine", "clone", "grow", "spin"], //"multiball"
-  // powerUpTypes : ["spin"],
+  // powerUpTypes : ["mine", "clone", "grow", "spin"], //"multiball"
+  powerUpTypes : ["mine"],
```
Comment (flukeout) [inline battle-pong/src/js/settings.js]: Revert plz

File: battle-pong/src/js/settings.js [hunk 2]
```diff
@@ -2,11 +2,11 @@ window.Settings = {
...
   brakesModeEnabled : false,
   goalTimeoutMS : 250,
-  powerupFrequency : 300, //300
+  powerupFrequency : 30, //300
```
Comment (flukeout) [inline battle-pong/src/js/settings.js]: Revert plz

**Pull request merged**